public/app/plugins/panel/nodeGraph/plugin.json

```diff
@@ -0,0 +1,16 @@
+{
+  "type": "panel",
+  "name": "Service Map",
+  "id": "serviceMap",
+
+  "info": {
+    "author": {
+      "name": "Grafana Labs",
+      "url": "https://grafana.com"
+    },
+    "logos": {
+      "small": "img/icn-logs-panel.svg",
+      "large": "img/icn-logs-panel.svg"
```

Comment:
Need to talk to Song and get a new panel icon designed. Need to mark this panel as alpha for now if you intend to merge it before that,